code/__DEFINES/colors.dm

```diff
@@ -182,13 +181,39 @@
 
 /// Starlight!
 #define COLOR_STARLIGHT "#8589fa"
+
 /**
  * Some defines to generalise colours used in lighting.
  *
  * Important note: colors can end up significantly different from the basic html picture, especially when saturated
+ *
  */
-/// Bright light used by default in tubes and bulbs
-#define LIGHT_COLOR_DEFAULT "#f3fffa"
+
+// Constructable fixtures and their subtypes
+
+/// Default in tube fixtures. Bright white with a slight green tint. rgb(243, 255, 250)
+#define LIGHT_FIXTURE_TUBE_NEUTRAL "#F3FFFA"
+/// Default in tubes, cold subtypes. Bright white with a slight blue tint.rgb(222, 239, 255)
+#define LIGHT_FIXTURE_TUBE_COLD "#DEEFFF"
+/// Default in tubes, warm subtypes. White with unsaturated orange. rgb(250, 229, 193)
+#define LIGHT_FIXTURE_TUBE_WARM "#FAE5C1"
+/// Default in night shift mode. Warm white with a slight pink tint. rgb(255, 201, 204)
+#define LIGHT_FIXTURE_NIGHT_SHIFT "#FFDDCC"
+/// Default in bulbs. Smaller AND warmer! rgb(255, 214, 170)
+#define LIGHT_FIXTURE_BULB_WARM "#FFD6AA"
+/// Less saturated red. rgb(255, 50, 50)
+#define LIGHT_FIXTURE_RED "#FF3232"
+/// Did you know we have two vars for fire colour? rgb(255, 17, 0)
+#define LIGHT_FIXTURE_RED_FIRE "#FF1100"
+/// For emergencies only, no fires allowed! rgb(255, 78, 78)
+#define LIGHT_FIXTURE_EMERGENCY "#FF4E4E"
+/// Rave party or crime scene. rgb(167, 0, 255)
+#define LIGHT_FIXTURE_BLACKLIGHT "#A700FF"
+/// Why are you still partying, there's a fire! rgb(212, 0, 255)
+#define LIGHT_FIXTURE_BLACKLIGHT_FIRE "#D400FF"
```

Comment:
These names are a bit weird, think it's better to describe the color rather then the particular usage, least that's how I've worked with it in the past.